config-solaris.md

```diff
@@ -12,15 +12,15 @@ The SMF(Service Management Facility) FMRI which should go to "online" state befo
 "milestone": "svc:/milestone/container:default"
 ```
 
-## <a name="configSolarisLimitpriv" />limitpriv
+## <a name="configSolarisLimitPriv" />limitpriv
```

Comment:
If you're changing `limitpriv` to `limitPriv` below, I expect you want to change it in the header here as well.  You've currently changed the anchor, but not the visible text.